src/algebra/jordan/basic.lean

```diff
@@ -0,0 +1,210 @@
+/-
+Copyright (c) 2021 Christopher Hoskin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Christopher Hoskin
+-/
+import algebra.ring.basic
+import algebra.lie.of_associative
+import data.real.basic
+import linear_algebra.basic
+
+/-!
+# Jordan algebras
+
+We define a set of conditions, formulated in terms of commuting multiplication operators, for the
+multiplication on a non-unital, non-associative semiring to be a Jordan multiplication. When the
+multiplication is commutative, these take a particularly simple form.
+
+A real Jordan algebra `A` can be introduced by
+```
+variables {A : Type*} [non_unital_non_assoc_ring A] [module ℝ A] [smul_comm_class ℝ A A]
+  [is_scalar_tower ℝ A A] [comm_jordan A]
+```
```

Comment:
Could you please expand this a bit, by explaining what a Jordan algebra is? Something similar to the intro paragraphs of https://en.wikipedia.org/wiki/Jordan_algebra would be quite helpful I think.

Reply:
Done. I also gave some indication of why one might want to study them.